code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -148,20 +153,21 @@
 		return
 	user << "<span class='shadowling'>You freeze the nearby air.</span>"
 	for(var/turf/T in targets)
-		for(var/mob/living/carbon/M in T.contents)
+		for(var/mob/living/carbon/human/M in T.contents)
 			if(is_shadow_or_thrall(M))
 				if(M == user) //No message for the user, of course
 					continue
 				else
 					M << "<span class='danger'>You feel a blast of paralyzingly cold air wrap around you and flow past, but you are unaffected!</span>"
 					continue
-			M << "<span class='userdanger'>A wave of shockingly cold air engulfs you!</span>"
-			M.Stun(2)
-			M.apply_damage(10, BURN)
-			if(M.bodytemperature)
-				M.bodytemperature -= 200 //Extreme amount of initial cold
-			if(M.reagents)
-				M.reagents.add_reagent("frostoil", 15) //Half of a cryosting
+			if(istype(M.wear_suit))
+				if(M.wear_suit.type in suits)
```

Comment:
Instead of this snowflakey check with that suits list,keep in mind that clothing has a min_cold_protection_temperature var, which spacesuits have it set to min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT . Check for this value since it handles cold protection